.github/workflows/test.yml

```
@@ -242,6 +265,9 @@ jobs:
       fail-fast: false
       matrix:
         include: ${{ fromJson(needs.discover.outputs.groups) }}
+    env:
+      LOTUS_RUN_EXPENSIVE_TESTS: 1
+      LOTUS_RUN_VERY_EXPENSIVE_TESTS: ${{ needs.discover.outputs.run_very_expensive_tests }}
```

We could also do this to be more concise, but I think it's just a matter of preference so I think we can leave that for the repo owners to decide.

```suggestion
      LOTUS_RUN_VERY_EXPENSIVE_TESTS: ${{ github.event_name == 'schedule' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'need/very-expensive-tests')) && '1' || '0' }}
```

---

Yup, I'm on team "readability counts",
@rvagg that one will be up to you

---

I like explicitness and readability so I'm fine with the more verbose form. Unfortunately the combination of Bash and GHA YAML make it kind of obscure whichever way you carve this (in those cases, comments can be a good substitute IMO).

---

Thanks for the input, I kept the verbose version and added documentation